examples/options/include/traccc/options/generation.hpp

```diff
@@ -42,6 +42,11 @@ class generation : public interface {
     opts::value_array<float, 2> phi_range{-180.f, 180.f};
     /// Range of eta
     opts::value_array<float, 2> eta_range{-2.f, 2.f};
+    /// Range of theta [rad] (corresponding to eta range of [-2,2]
+    opts::value_array<float, 2> theta_range{
+        15.415f * detray::unit<float>::degree,
+        164.585f * detray::unit<float>::degree};
```

Comment:
To be absolutely sure that we do the right thing, how about:

```suggestion
    opts::value_array<float, 2> theta_range{
        eta_to_theta_range(eta_range)};
```

?

Reply:
OK. I also changed the CMake `traccc_performance` to the public link to make this work